website/src/data/users.js

```diff
@@ -258,6 +258,15 @@ const users = [
     fbOpenSource: false,
     pinned: true,
   },
+  {
+    title: 'MateuszDabrowski',
+    description: 'Salesforce Marketing Cloud Programmatic Documentation',
```

Comment:
Is this description accurate? It's perfectly okay to describe this site as your personal branding site and we rather give accurate descriptions.

Reply:
I was thinking about the proper description and wanted it to showcase the main value behind the page for the potential viewer. I might propose an updated version with `'Personal branding site with unofficial Salesforce Marketing Cloud Programmatic Documentation'`. I believe that the main content on the site is focused on JS/SSJS and soon SFMC SQL docs, not personal items. Let me know if you would prefer the description proposed above.